super_team_picker/index.js

```diff
@@ -0,0 +1,151 @@
+const express = require('express');
+const path = require('path');
+const methodOverride = require('method-override');
+const knex = require('./db/client');
+
+const app = express();
+
+app.use(express.urlencoded({extended: true}));
+app.use(methodOverride('_method'));
+app.use(express.static(path.join(__dirname, 'public')));
+app.set('view engine', 'ejs');
+app.set('views', path.join(__dirname, '/views'));
+
+
+
+app.get('/', (req, res)=> {
+    res.render('home')
+})
+
+app.get('/cohorts/new', (req, res) => {
```

Comment:
Hi teacher. Yes. I know I can create a router for cohorts. But that is not one of the homework's requirements. Also this app is only about cohorts, nothing else. So '/cohorts' is like part of the base url. Putting them in the index.js file makes 100% sense in my opinion. I'm just being lazy and believe adding a router for cohorts is neither necessary nor required. Did I lost 1% mark because of this? If that's the case, I'm greedy and I want to argue my mark lol.

Reply:
@PinkSummerSea  No, this is not the reason for 1% deduction but it is definitely a good practice to follow especially if this is something you hardly see in real practice. I pointed it out so that you don't do this next time thus making your code more dry. FYI, for any homework I mark, I don't deduct marks because code is repetitive, but because logic is incorrect or a feature mentioned in the rubric is not working.

1% deduction is due to how similar your ui is to the mocks. Basically, part 3 is out of 6 marks and I gave you 5. Once the total mark is calculated out of 100% based on the marking scheme, it came out to be 99.

Good thing I see though from marking 4 assignments so far is that you are actually paying attention to my comments and writing code that is dry and easy to read, which I really appreciate especially when I'm taking my time to read through your code. So thank you for that and keep up the good work 👍

Reply:
Thank you for quick reply! So the 1% reduction is because my ui is not similar enough to the mocks, right? That makes sense cuz I know I didn't manage to make them 100% identical haha. I also appreciate your detailed comments for each assignment. I'm learning good practice from your comments. By the way, submitting code on moodle is required by our instructor, which she said is for record purpose only.

Reply:
Oh, did she say to submit ALL the files? If that's the case, ignore my comment for that. From what I'm told, I thought just a single txt file with PR link is good enough rather than 15 files of js files. But okay!

Reply:
Yeah she said all the files except node_modules.